hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -107,27 +100,61 @@ def hpat_pandas_series_getitem_idx_series_impl(self, idx):
 
 @overload_attribute(SeriesType, 'at')
 @overload_attribute(SeriesType, 'iat')
+def hpat_pandas_series_iat(self, idx):
+    """
+    Pandas Series operators :attr:`pandas.Series.at`, :attr:`pandas.Series.iat` implementation.
+    .. only:: developer
+
+        Tests: python -m hpat.runtests -k hpat.tests.test_series.TestSeries.test_at_series*
+        Tests: python -m hpat.runtests -k hpat.tests.test_series.TestSeries.test_iat_series*
+
+    Parameters
+    ----------
+    self: :obj:`pandas.Series`
+        input series
+
+    Returns
+    -------
+    :obj:`pandas.Series`
+        returns an object of :obj:`pandas.Series`
+    """
+
+    _func_name = 'Operator at/iat().'
+
+    if not isinstance(self, SeriesType):
+        raise TypingError('{} The object must be a pandas.series. Given: {}'.format(_func_name, self))
+
+    if not isinstance(idx, types.Integer):
+        raise TypingError('{} The index must be an Integer. Given: {}'.format(_func_name, idx))
+
+    def hpat_pandas_series_iat_impl(self):
+        return self
```

Comment:
Why you implement separate method if the realization (`return self`) is the same?

Reply:
I want to rewrite getitem() so that I can get the item by index, not just by id, to implement methods at/iat/loc/iloc like in pandas